src/cloudWatchLogs/commands/saveCurrentLogDataContent.ts

```diff
@@ -49,8 +44,8 @@ export async function saveCurrentLogDataContent(uri: vscode.Uri | undefined, reg
 
         if (selectedUri) {
             try {
-                await fs.writeFile(selectedUri.fsPath, content)
-                // TODO: Open file and close virtual doc? Is this possible?
+                const bytes = new TextEncoder().encode(content)
+                await vscode.workspace.fs.writeFile(selectedUri, bytes)
```

Comment:
Use the new fs class which does this under the hood

```suggestion
                await FileSystemCommon.instance.writeFile(selectedUri, content)
```
```